src/main/java/moimoi/util/task/Deadline.java

```diff
@@ -0,0 +1,62 @@
+package moimoi.util.task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+
+    protected LocalDateTime by;
```

Comment:
Perhaps the name could be better, for example maybe we can use deadline as the name so that we know what the variable is :)

Reply:
Thanks! I agree with your feedback👍 I'll tweak my code accordingly to improve variable readability.